OctopusDSC/Tests/cOctopusServer.Tests.ps1

```diff
@@ -189,151 +324,153 @@ try
                 }
             }
 
-            function Get-CurrentConfiguration ([string] $testName) {
-                & (Resolve-Path "$PSCommandPath/../../Tests/OctopusServerExeInvocationFiles/$testName/CurrentState.ps1")
-            }
-
-            function Get-RequestedConfiguration ([string] $testName) {
-                & (Resolve-Path "$PSCommandPath/../../Tests/OctopusServerExeInvocationFiles/$testName/RequestedState.ps1")
-            }
+            Context "Octopus server command line" {
```

Comment:
Mainly indenting that has confused git...

Reply:
Git seems very confused about this diff generally.